### ISSUE TITLE
Trailing Slash in basePath and leading slash in route path leads to double slash in URL

### ISSUE THREAD

**Issue by gshackles** [fsprojects/SwaggerProvider#65]
### Description

When the Swagger input has a `basePath` with a trailing slash, and a path with a leading slash, the resulting concatenated URL will end up with two slashes. While this is what was technically specified in the input, it's a good chance to be a little more flexible and remove one of the slashes from the resulting URL.

### Repro steps

Use a Swagger input that looks something like:

```
{
  "host": "api.host.com",
  "basePath": "/",
  "paths": {
    "/v0.1/user": {
```

### Expected behavior

The URL would be `https://api.host.com//v0.1/user`

### Actual behavior

The URL would be `https://api.host.com/v0.1/user`

**Comment by sergey-tihon** [comment]
@gshackles What is wrong with `//` in url?
I always thought that servers process such requests correctly.

**Comment by gshackles** [comment]
Depends on the server I think, last night I ran into this with the VS Mobile Center API where the extra slash broke things

**Comment by sergey-tihon** [comment]
Should be fixed in [0.5.7](https://www.nuget.org/packages/SwaggerProvider/0.5.7)
